sklearn/metrics/tests/test_pairwise.py

```diff
@@ -484,13 +484,24 @@ def check_pairwise_distances_chunked(X, Y, working_memory, metric='euclidean'):
     S = pairwise_distances(X, Y, metric=metric)
     assert_array_almost_equal(blockwise_distances, S)
 
+@ignore_warnings
```

Comment:
What warnings are you ignoring? Rather, you should be asserting the absence of any warning.